mito-ai/mito_ai/completions/prompt_builders/agent_system_message.py

```diff
@@ -82,7 +85,8 @@ def create_agent_system_message_prompt(isChromeBrowser: bool) -> str:
 1. The index should be the 0-index position of where you want the new code cell to be added in the notebook.
 2. The message is a short summary of your thought process that helped you decide what to update in cell_update.
 3. The code should be the full contents of that updated code cell. The code that you return will overwrite the existing contents of the code cell so it must contain all necessary code.
-4. The cell_type should only be 'markdown' if there is no code to add. There may be times where the code has comments. These are still code cells and should have the cell_type 'code'. Any cells that are labeled 'markdown' will be converted to markdown cells by the user.
+4. The code_summary is a one sentence summary of the code that you are adding. It should be no more than 5 words, and start with a verb. For example, "Converting the transaction_date column", "Calculating the all time high closing price", "Plotting total_price", etc.
```

Comment:
"Calculating the all time high closing price" more than 5 words :) Maybe we want to be more lenient than 5?